src/AppInstallerCLICore/ExecutionArgs.h

```diff
@@ -81,6 +81,7 @@ namespace AppInstaller::CLI::Execution
 
             // Other
             All, // Used in Update command to update all installed packages to latest
+        	ListAll, // Used in List command to list all installed apps associated with a package

```

Comment:
Why do you need to add a new arg here? Couldn't you reuse `AppInstaller::CLI::Execution::Args::Type::All` ?

Reply:
`AppInstaller::CLI::Execution::Args::Type::All` is used in `winget upgrade --all`, where I wanted the simplified table to print out. Also, there's a different help string for this new argument, which would mean I'd have to have the Argument defined only in ListCommand instead of in the Arguments.cpp file with the rest of them...